internal/kafka/internal/config/kafka.go

```diff
@@ -11,13 +11,14 @@ import (
 )
 
 type KafkaCapacityConfig struct {
-	IngressEgressThroughputPerSec string `yaml:"ingressEgressThroughputPerSec"`
-	TotalMaxConnections           int    `yaml:"totalMaxConnections"`
-	MaxDataRetentionSize          string `yaml:"maxDataRetentionSize"`
-	MaxPartitions                 int    `yaml:"maxPartitions"`
-	MaxDataRetentionPeriod        string `yaml:"maxDataRetentionPeriod"`
-	MaxConnectionAttemptsPerSec   int    `yaml:"maxConnectionAttemptsPerSec"`
-	MaxCapacity                   int64  `yaml:"maxCapacity"`
+	IngressThroughputPerSec     string `json:"ingressThroughputPerSec"`
+	EgressThroughputPerSec      string `json:"egressThroughputPerSec"`
+	TotalMaxConnections         int    `json:"totalMaxConnections"`
+	MaxDataRetentionSize        string `json:"maxDataRetentionSize"`
+	MaxPartitions               int    `json:"maxPartitions"`
+	MaxDataRetentionPeriod      string `json:"maxDataRetentionPeriod"`
+	MaxConnectionAttemptsPerSec int    `json:"maxConnectionAttemptsPerSec"`
+	MaxCapacity                 int64  `json:"maxCapacity"`
```

Comment:
just a note, this MaxCapacity isn't actually used anymore, will create a PR against main to remove it.

Reply:
looks like this is not in main but in the larger-kafka-instances branch. Created PR to remove this in the feature branch: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/863